backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc

```diff
@@ -45,6 +45,8 @@ include::modules/oadp-installing-dpa-1-2-and-earlier.adoc[leveloffset=+1]
 include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 include::modules/oadp-creating-object-bucket-claim.adoc[leveloffset=+2]
 include::modules/oadp-enabling-csi-dpa.adoc[leveloffset=+2]
+include::modules/oadp-about-disable-node-agent-dpa.adoc[leveloffset=+2] 
+For more details, see xref:../../../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-jobs[Running tasks in pods using jobs].
```

Comment:
Same comment: This link belongs in the additional resources section.

Reply:
applied the change